.eslintrc.yml

```diff
@@ -14,6 +14,10 @@ parserOptions:
 plugins:
   - '@typescript-eslint'
 rules:
+  no-unused-vars:
+    - off
```

Comment:
What is the point of turning this of? just for curiosity

Reply:
That rule is for javascript projects, and conflicts with the typescript-eslint plugin rule. I disable it to use the typescript one